swagger.json

```diff
@@ -2574,9 +2574,234 @@
                     }
                 }
             }
+        },
+        "/addresses": {
+            "get": {
+                "tags": [
+                    "Default", "Fleet"
+                ],
+                "summary": "/addresses",
+                "description": "Fetch all addresses/geofences for the organization",
+                "operationId": "GetOrganizationAddresses",
+                "parameters": [
+                    { "$ref": "#/parameters/accessTokenParam" }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "List of addresses/geofences. For each address entry, the geofence field will only be populated with either 'circle' or 'polygon', depending on the geofence type.",
+                        "schema": {
+                            "type": "array",
+                            "items": {
+                                "$ref": "#definitions/Address"
+                            }
+                        }
+                    },
+                    "default": {
+                        "description": "Unexpected error.",
+                        "schema": {
+                            "$ref": "#/definitions/ErrorResponse"
+                        }
+                    }
+                }
+            },
+            "post": {
+                "tags": [
+                    "Default", "Fleet"
+                ],
+                "summary": "/addresses",
+                "description": "Add one or more addresses to the organization",
```

Comment:
I believe this endpoint only allows creating a single address entry.

Reply:
As described in the spec, this takes in an array of addresses (so they could add one with an array of size one)

Reply:
Ok, please ignore this comment then :)